app/publish/testing.go

```diff
@@ -0,0 +1,38 @@
+package publish
+
+import (
+	"bytes"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"testing"
+
+	"github.com/lbryio/lbrytv/internal/lbrynet"
+
+	"github.com/stretchr/testify/require"
+)
+
+// CreatePublishRequest creates and returns a HTTP request providing data for the publishing endpoint.
+func CreatePublishRequest(t *testing.T, data []byte) *http.Request {
+	readSeeker := bytes.NewReader(data)
+	body := &bytes.Buffer{}
+
+	writer := multipart.NewWriter(body)
+
+	fileBody, err := writer.CreateFormFile(FileFieldName, "lbry_auto_test_file")
+	require.Nil(t, err)
```

Comment:
I like this! I need to start using this instead.

Reply:
Yeah, it's quite handy. One problem is it doesn't handle printing `lbry.go` custom errors very well (I mentioned that before). I'll get around to fixing it one day.